server/models/order.js

```diff
@@ -0,0 +1,2 @@
+const orders = [];
+export default orders;
```

Comment:
The default export should be the same name as the file. Change `order.js` to `orders.js`.

Reply:
@andrewinsoul. Please address this issue so we can merge this PR.